services/QuillLMS/app/views/pages/admins.html.erb

```diff
@@ -1,2 +1,181 @@
+<% for_administrators_img_base = "#{ENV['CDN_URL']}/images/pages/for_administrators" %>
```

Comment:
Lines 1-6 could be in a helper or controller

Reply:
They could, but I feel like they're view-specific enough that to me it feels easier to keep them in this file. If you or other feel strongly, though, I can make a change.

Reply:
My impetus for the suggestion is not that these lines aren't view-specific, but that any logic that can be reasonably moved out of views is generally a good thing - easier unit testing, for example. But variable assignment is arguably not logic, so I was on the fence. Thus, I approved the PR and consider this an optional enhancement.